Resources/views/CRUD/edit_orm_many_to_many.html.twig

```diff
@@ -9,7 +9,8 @@ file that was distributed with this source code.
 
 #}
 {% if sonata_admin.field_description.hasassociationadmin %}
-    <div id="field_container_{{ id }}" class="field-container">
+    <div id="field_container_{{ id }}"
+		class="field-container {{ form.vars["attr"]["class"] is defined ? form.vars["attr"]["class"] }}">
```

Comment:
`""` should be used only when string interpolation is needed (it helps noticing it when all the rest of the code uses single quotes as enclosures)